projects/demo/src/modules/components/alert/index.ts

```diff
@@ -20,7 +19,7 @@ import {AlertExampleWithDataComponent} from './examples/4';
 
 @Component({
     standalone: true,
-    imports: [TuiDemo, TuiLink, TuiButton, TuiTextCodeDirective, RouterLink],
+    imports: [TuiDemo, TuiLink, TuiButton, TuiTextCode],
```

Comment:
Are you sure router link is not used?
https://github.com/taiga-family/taiga-ui/blob/main/projects/demo/src/modules/components/alert/index.html#L129
Please check all the places where you removed it

Reply:
RouterLink included inside TuiDemo